lib/typescript/render/outbound/chatplugin.ts

```diff
@@ -10,4 +11,17 @@ export class ChatpluginMapper extends OutboundMapper {
   isTextSupported(): boolean {
     return true;
   }
+
+  getAttachmentPayload(mediaUrl: string): any {
+    const mediaType = getAttachmentType(mediaUrl, 'chatplugin');
+
+    return {
+      attachment: {
+        type: mediaType,
+        payload: {
+          url: mediaUrl,
+        },
+      },
+    };
+  }
 }
```

Comment:
added `getAttachmentPayload` to the Chatplugin outbound mapper to be able to use the OutboundMapper type (but obviously this should be completed with your PR @thorstenairy)